pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/BundleStats.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.policies.data;
+
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class BundleStats {
+    public List<String> topics;
+
+    public String bundleName;
+    public double msgRateIn;
+    public double msgThroughputIn;
+    public double msgRateOut;
+    public double msgThroughputOut;
+    public int consumerCount;
+    public int producerCount;
+    public long topicCount;
```

Comment:
Nit: this field is `topics` in the `NamespaceBundleStats`. I think it is better to use `topicCount`, and I don't believe `NamespaceBundleStats` is used externally. So, I think this divergence is fine. However, I want to call this out for any others reviewing this PR.